curriculum/challenges/english/03-front-end-libraries/react/review-using-props-with-stateless-functional-components.english.md

```diff
@@ -23,15 +23,15 @@ The code editor has a <code>CampSite</code> component that renders a <code>Campe
 ```yml
 tests:
   - text: The <code>CampSite</code> component should render.
-    testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(CampSite)); return mockedComponent.find('CampSite').length === 1; })(), 'The <code>CampSite</code> component should render.');
+    testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(CampSite)); return mockedComponent.find('CampSite').length === 1; })());
   - text: The <code>Camper</code> component should render.
-    testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(CampSite)); return mockedComponent.find('Camper').length === 1; })(), 'The <code>Camper</code> component should render.');
+    testString: assert((function() { const mockedComponent = Enzyme.mount(React.createElement(CampSite)); return mockedComponent.find('Camper').length === 1; })());
   - text: The <code>Camper</code> component should include default props which assign the string <code>CamperBot</code> to the key <code>name</code>.
-    testString: getUserInput => assert((function() { const noWhiteSpace = getUserInput('index').replace(/\s/g, ''); const verify1 = 'Camper.defaultProps={name:\'CamperBot\'}'; const verify2 = 'Camper.defaultProps={name:"CamperBot"}'; return (noWhiteSpace.includes(verify1) || noWhiteSpace.includes(verify2)); })(), 'The <code>Camper</code> component should include default props which assign the string <code>CamperBot</code> to the key <code>name</code>.');
+    testString: getUserInput => assert((function() { const normalized = getUserInput('index').replace(/\s/g, '').replace(/"/g,'\''); const verify1 = 'Camper.defaultProps={name:\'CamperBot\'}'; const verify2 = 'Camper.defaultProps={name:\'CamperBot\',}'; return (normalized.includes(verify1) || normalized.includes(verify2)); })());
```

Comment:
Why not just use a regex and do away with normalization and two verify consts?

Reply:
Thanks for the review!  The reasoning for not using regex was essentially that the change to the functionality was tiny.  It seemed okay to make a small change rather than a complete rewrite.  However, if you reckon regex would be easier to maintain, I'm happy to change it.

Reply:
Now the tested strings are practically identical, so that's why a regex makes sense. Also then there's no need to normalize the quotes.